tests/components/raycaster.test.js

```diff
@@ -80,8 +80,10 @@ suite('raycaster', function () {
       var newEl = this.targetEl = document.createElement('a-entity');
       var numObjects = el.components.raycaster.objects.length;
       newEl.addEventListener('loaded', function () {
-        assert.equal(el.components.raycaster.objects.length, numObjects + 1);
-        done();
+        process.nextTick(function () {
```

Comment:
Saw the test was failing. Maybe the raycaster handler was getting called after this handler.